vault/resource_kv_secret_test.go

```diff
@@ -69,3 +75,46 @@ resource "vault_kv_secret" "test" {
 
 	return ret
 }
+
+func testKVSecretConfigUpdate(mount, name string) string {
```

Comment:
We can rename this to `testKVSecretConfig_updated` and also rename  `testKVSecretConfig` -> `testKVSecretConfig_basic` on L59; there is precedent in the codebase for naming config functions in that style :)

Reply:
I committed the renaming.